app/models/spree/user_decorator.rb

```diff
@@ -0,0 +1,5 @@
+Spree::User.class_eval do
+  def full_name
```

Comment:
Yeah, it gets called in the model decorators when they define `who=(user_id)` Here is an example: https://github.com/solidusio-contrib/solidus_papertrail/blob/4effa097c6ab3d574182e9b184ec01c6db271a4e/app/models/spree/adjustment_decorator.rb#L9